tests/test_0414-write-jagged-arrays.py

```diff
@@ -200,15 +200,15 @@ def test_awkward_jagged_metadata(tmp_path):
 
     b1 = t1.GetBranch("b1")
     assert b1.GetLeaf("b1").GetName() == "b1"
-    assert b1.GetLeaf("b1").GetLeafCount() == None  # noqa: E711 (ROOT null check)
+    assert b1.GetLeaf("b1").GetLeafCount() is None
```

Comment:
Wow, this really is an object that has been overloaded to compare with `==` to None? That is really ugly on ROOT's side! Can't we just compare truthiness here? Using `== None` is really a anti-Pythonism.